automated install/basic-install.sh

```diff
@@ -1343,8 +1351,9 @@ installConfigs() {
     # Make sure Pi-hole's config files are in place
     version_check_dnsmasq
     # Install empty file if it does not exist
-    if [[ ! -f "${PI_HOLE_CONFIG_DIR}/pihole-FTL.conf" ]]; then
-        if ! install -o pihole -g pihole -m 664 /dev/null "${PI_HOLE_CONFIG_DIR}/pihole-FTL.conf" &>/dev/null; then
+    if [[ ! -r "${PI_HOLE_CONFIG_DIR}/pihole-FTL.conf" ]]; then
+        install -d -m 0755 ${PI_HOLE_CONFIG_DIR}
+        if ! install -o pihole -m 664 /dev/null "${PI_HOLE_CONFIG_DIR}/pihole-FTL.conf" &>/dev/null; then
```

Comment:
How about `install -D` and then there's no need to have separate directory and file creation.

Reply:
But with a seperate command you can provide the permissions for the directory. With -D you don't really know what you end up. It depends on the default umask set.

Reply:
Lessons learned from RPM packaging ;-)